requirements.txt

```diff
@@ -0,0 +1,19 @@
+appdirs==1.4.4
```

Comment:
It might be worth investing in a more robust dependency management system, eg: https://python-poetry.org/

Although I guess since there's only one module in this project the gain is mostly educational... :D

Reply:
will implement next commit, ty for recommend